File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res = "";
        while(columnNumber){
            columnNumber--;
            char c = 'A'+columnNumber%26;
            res = c+res;
            columnNumber/=26;
        }
        return res;
    }
};

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
      String ans = "";
        while(columnNumber!=0){
            columnNumber--;
            int ch = 'A'+columnNumber%26;
            ans = (char)ch+ans;
            columnNumber = columnNumber/26;
        }
        return ans;
    }
}